packages/base/links-to-many-component.gts

```diff
@@ -122,13 +129,33 @@ interface LinksToManyStandardEditorSignature {
 class LinksToManyStandardEditor extends GlimmerComponent<LinksToManyStandardEditorSignature> {
   @consume(CardContextName) declare cardContext: CardContext;
 
+  @action
+  reorderItems(items: any) {
```

Comment:
Maybe a better name would be to `setItems` since there is no actual reordering going on here. It already receives the ordered items